android/src/ti/map/AnnotationProxy.java

```diff
@@ -275,8 +289,10 @@ private void handleImage(Object image)
 			TiDrawableReference imageref = TiDrawableReference.fromUrl(this, (String) image);
 			Bitmap bitmap = imageref.getBitmap();
 			if (bitmap != null) {
-				markerOptions.icon(BitmapDescriptorFactory.fromBitmap(bitmap));
-				setIconImageDimensions(bitmap.getWidth(), bitmap.getHeight());
+				try {
```

Comment:
Not directly related to this PR, but we could take advantage of `TiDrawableReference.fromObject` in the handling of the image source to streamline the method a bit. Something in the lines of :
```

// this will go through different image sources
TiDrawableReference imageRef = TiDrawableReference.fromObject(this, (String) image);
// check if we loaded the image succesfully
if (!(imageRef.isTypeNull())) {
    // get the bitmap
    Bitmap bitmap = imageref.getBitmap();
    // guard for the bitmap
    if (bitmap != null) {
        try {
            // use it     
            markerOptions.icon(BitmapDescriptorFactory.fromBitmap(bitmap));
            setIconImageDimensions(bitmap.getWidth(), bitmap.getHeight());
        catch (Exception e) {
            e.printStackTrace();
        }
        return;
    }
}
```

Reply:
I left this open, could be handled after the commit